htdocs/core/modules/facture/doc/pdf_crabe.modules.php

```diff
@@ -1850,14 +1850,16 @@ protected function _pagehead(&$pdf, $object, $showaddress, $outputlangs, $output
 
 
 			// Show sender frame
-			$pdf->SetTextColor(0, 0, 0);
-			$pdf->SetFont('', '', $default_font_size - 2);
-			$pdf->SetXY($posx, $posy - 5);
-			$pdf->MultiCell($widthrecbox, 5, $outputlangs->transnoentities("BillFrom"), 0, $ltrdirection);
-			$pdf->SetXY($posx, $posy);
-			$pdf->SetFillColor(230, 230, 230);
-			$pdf->MultiCell($widthrecbox, $hautcadre, "", 0, 'R', 1);
-			$pdf->SetTextColor(0, 0, 60);
+			if (empty($conf->global->MAIN_PDF_NO_SENDER_FRAME)) {
```

Comment:
Not that crabe is deprecated and must be replace with sponge. Can you add it also into sponge ?

Reply:
@eldy it's done